### DISCUSSION TITLE
Relative links in markdown are broken

### DISCUSSION THREAD

**Discussion by dominictobias** [npm/feedback#666] [Registry] [answered]
It's a shame that we can't make use of relative links that are supported by git hosts such as GitHub. On GitHub the following is valid:

```
[More Detailed Docs](docs/some-doc.md)
```

This also means links work on forked repos.

However, NPM will output this link as:

```
https://www.npmjs.com/package/@my-org/docs/some-doc.md
```

Note that it omits the package name there, which is also a bug?

Instead of using `homepage` if present in `package.json` as the base path:

<img width="730" alt="Screenshot 2022-04-10 at 23 39 27" src="https://user-images.githubusercontent.com/760314/162642951-b97f2c60-3788-4c74-9791-e3e1a70f2973.png">

**Comment by monishcm** [comment, marked as answer]
I've added your issue to our backlog. Thanks for reporting and we will investigate this.

**Reply by jsamr** [reply]
@monishcm Did the engineering team find time to investigate? Would you have any update to share?

**Comment by cybairfly** [comment]
I thought this was just a bug and cannot believe this is actually the standard behavior on npm. How can relative links work anywhere else but on npm where I publish the package? Is there any official solution? It's madness!!! ;-)
